### PR TITLE
chore: changing Safe core addresses for Base, Optimism, Celo

### PR DESCRIPTION
**Pull request by kupermind**
- Сhanging Safe core addresses for Base, Optimism, Celo;
- Goerli mentioning to legacy, except for tests for now;
- Updating packages and workflows;
- NO CODE CHANGE.

### CODE REVIEW

File: scripts/deployment/l2/globals_celo_mainnet.json [hunk 1]
```diff
@@ -1 +1 @@
...02ED2cA2d42104F621f448A2cE1bC8Fb7","olasAddress":"0xaCFfAe8e57Ec6E394Eb1b41939A8CF7892DbDc51","multisigProxyHash130":"0xb89c1b3bdf2cf8827818646bce9a8f6e372885f8c55e5c07acbd307cb133b000","serviceRegistryAddress":"0xE3607b00E75f6405248323A9417ff6b39B244b50","operatorWhitelistAddress":"0x3C1fF68f5aa342D296d4DEe4Bb1cACCA912D95fE","serviceRegistryTokenUtilityAddress":"0x3d77596beb0f130a4415df3D2D8232B3d3D31e44","serviceManagerTokenAddress":"0x34C895f302D0b5cf52ec0Edd3945321EB0f83dd5","gnosisSafeMultisigImplementationAddress":"0x63e66d7ad413C01A7b49C7FF4e3Bb765C4E4bd1b","gnosisSafeSameAddressMultisigImplementationAddress":"0xBb7e1D6Cb6F243D6bdE81CE92a9f2aFF7Fbe7eac","stakingTokenAddress":"0xe1E1B286EbE95b39F785d8069f2248ae9C41b7a9","stakingNativeTokenAddress":"","minStakingDepositLimit":"10000000000000000000000","timeForEmissionsLimit":"2592000","numServicesLimit":"100","apyLimit":"3000000000000000000","stakingVerifierAddress":"0xc40C79C275F3fA1F3f4c723755C81ED2D53A8D81","stakingFactoryAddress":"0x1c2cD884127b080F940b7546c1e9aaf525b1FA55","wormholeL1MessageRelayerAddress":"0x27428DD2d3DD32A4D7f7C497eAaa23130d894911","wormholeMessengerAddress":"0x397125902ED2cA2d42104F621f448A2cE1bC8Fb7"}
```
Comment (kupermind) [inline scripts/deployment/l2/globals_celo_mainnet.json:1]: [GnosisSafe](https://github.com/safe-global/safe-deployments/blob/eb2be45b484749a77a9f26c9c5e3b800a717115b/src/assets/v1.3.0/gnosis_safe.json#L218) and [GnosisSafeProxyFactory](https://github.com/safe-global/safe-deployments/blob/eb2be45b484749a77a9f26c9c5e3b800a717115b/src/assets/v1.3.0/proxy_factory.json#L218) contracts from eip155 to canonical

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -1,42 +0,0 @@
-name: Release Flow
```
Comment (kupermind) [inline .github/workflows/release.yaml:1]: Legacy, not used

File: .github/workflows/workflow.yaml [hunk 3]
```diff
@@ -20,10 +20,8 @@ jobs:
         with:
           submodules: recursive
 
-      - name: Get a specific Node.js version such that hardhat does not hang periodically
```
Comment (kupermind) [inline .github/workflows/workflow.yaml:23]: Other changes are script-package related.

**Review by DavidMinarsch — APPROVED** [review]

**Review by 77ph — COMMENTED** [review]
lgtm

**Pull request merged**